src/ceph.in

```diff
@@ -890,14 +896,16 @@ def main():
             timeout = 5
 
         hdr('Monitor commands:')
-        print('[Contacting monitor, timeout after %d seconds]' % timeout)
+        if verbose:
+            print('[Contacting monitor, timeout after %d seconds]' % timeout)
+        return do_extended_help(parser, childargs, ('mon', ''), ' '.join(childargs))
```

Comment:
oh, sorry. i think it's an improvement.  ceph always prints `[Contacting monitor, timeout after 5 seconds]` when user typed `ceph --help`, as following:

```
[root@node44 /data/seal/deploy]# ceph osd --help
[a lot of help messages]

 Monitor commands:
 =================
[Contacting monitor, timeout after 5 seconds]
2017-06-02 11:55:28.437591 7f49598db700 -1 WARNING: the following dangerous and experimental features are enabled: bluestore,rocksdb
auth add <entity> {<caps> [<caps>...]}   add auth info for <entity> from input
                                          file, or random key if no input is
                                          given, and/or any caps specified in
                                          the command
```